File: abc/abc182/a.java
// https://atcoder.jp/contests/abc182/submissions/41453750

import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner stdIn = new Scanner(System.in);

    int a = stdIn.nextInt();
    int b = stdIn.nextInt();

    if(2 * a + 100 <= b) {
      System.out.println(0);
    } else {
      System.out.println(2 * a + 100 - b);
    }
  }
}

File: abc/abc182/a.cs
// https://atcoder.jp/contests/abc182/submissions/46505910

using System;

public class A
{
  static void Main()
  {
    string[] t = Console.ReadLine().Split(" ");
    int a = int.Parse(t[0]);
    int b = int.Parse(t[1]);

    Console.Write(2 * a + 100 - b);
  }
}

